lib/cartopy/crs.py

```diff
@@ -3219,6 +3219,59 @@ def x_limits(self):
     def y_limits(self):
         return self._y_limits
 
+class Spilhaus(Projection):
+    """
+    Spihaus-Adams World Ocean Map in a Square II
+
+    """
+    def __init__(self,greenland_at = 0,false_easting = 0.0,false_northing = 0.0):
+        """
+        Parameters
+        ----------
+        greenland_at: optional
+            Orientation of the map. 0,1,2,3, stand for Greenland at upper left,
+            upper right, lower right and lower left, respecivly. Defaults to 0.
+            Strings like 'upper left' are also allowed.
+        false_easting: optional
+            X offset from the planar origin in metres. Defaults to 0.0.
+        false_northing: optional
+            Y offset from the planar origin in metres. Defaults to 0.0.
+        """
+        allowed_ori = ['upper left', 'upper right', 'lower right','lower left']
+        if isinstance(greenland_at, str):
+            if greenland_at in allowed_ori:
+                greenland_at = allowed_ori.index(greenland_at)
+            else:
+                raise ValueError('The orientation of Greenland must be one'
+                    ' of {allowed_ori}, or integer [0,1,2,3]')
+        if isinstance(greenland_at, int) and (0<=greenland_at<4):
+            rot_spil = 45+(greenland_at%4)*90
+        else:
+            raise ValueError('The orientation of Greenland must be one'
+                ' of {allowed_ori}, or integer [0,1,2,3]')
+        proj4_params = [('proj', 'spilhaus'),
+                        ('lat_0', -49.56371678),
+                        ('lon_0', 66.94970198),
+                        ('azi',40.17823482),
+                        ('k_0', 1.4142135623730951),
```

Comment:
We shouldn't hard-code any of the defaults being passed into PROJ, if they choose to change those defaults later I think we'd want to just go along with them. So in the proj4_params we should only pass in the values that the user has the options to change.